axon-framework/deadlines/deadline-managers.md

```diff
@@ -116,3 +121,26 @@ public void handle(PublishTime cmd) {
 ```
 
 Note that the current timestamp is automatically added to the EventMessage. If handlers only need to rely on the timestamp the event was published, they can access that timestamp directly, as described in [Handling Events](../events/event-handlers.md).
+
+> **Spring Configuration**
```

Comment:
I think the notation of this section would be nicer to make it an actual section instead of a comment.

So, adjust the following line like so:
```suggestion
## Configuration
```

And, remove all the `>` before every sentence.

Next to adjust the way it's provided, I would also provide a non-Spring configuration approach. So, through using the `Configurer#registerComponent` method in something like this:
```java
public class AxonConfig {
    // omitting other configuration methods...
    public void registerDeadlineManager(Configurer configurer, JobScheduler scheduler) {
        configurer.registerComponent(
                DeadlineManager.class,
                config -> JobRunrDeadlineManager.builder()
                                                .jobScheduler(scheduler)
                                                .scopeAwareProvider(config.scopeAwareProvider())
                                                .serializer(config.eventSerializer())
                                                .transactionManager(config.getComponent(
                                                        TransactionManager.class
                                                ))
                                                .spanFactory(config.spanFactory())
                                                .build()
        );
    }
}
```